autodepgraph/graph.py

```diff
@@ -330,8 +334,8 @@ def draw_mpl(self, ax=None):
         else:
             pos = self._generate_node_positions(node_positions)
         nx.draw_networkx_nodes(self, pos, ax=ax, node_color=colors_list)
-        nx.draw_networkx_edges(self, pos, ax=ax, arrows=True)
-        nx.draw_networkx_labels(self, pos, ax=ax)
+        nx.draw_networkx_edges(self, pos, ax=ax, arrows=True, **self.matplotlib_edge_properties)
```

Comment:
It clearly uses the defaults specified up top but they seem hardcoded. Could you motivate why one would want to do this?